gcp_variant_transforms/libs/bigquery_row_generator_test.py

```diff
@@ -295,9 +305,9 @@ def test_nonstandard_float_values(self):
     variant = vcfio.Variant(
         reference_name='chr19', start=11, end=12, reference_bases='CT',
         alternate_bases=[], filters=[],
-        info={'II': vcfio.VariantInfo(float('inf'), '1'),
```

Comment:
@arostamianfar I am not sure what the original reason for these type mismatches were, i.e., II* fields are TYPE_INTEGER while the values here are float. With the changes in this PR, the original version of this test fails as expected and I changed these to TYPE_FLOAT because the case that I think needs to be covered is this (in particular the one based on gnomAD, reported in Issue #48 as well).

Please comment if the original version of this test was also based on some realistic cases that we want to cover.

Reply:
ah, thanks for fixing this! This was an oversight from my end not realizing that `float(maxint) != maxint`. I'm a bit uneasy about using float(maxint) as well as it becomes a strange value when loaded to BQ (`9.223372036854776e+18`). What do you think of using a smaller value that still makes `float(val) == val`?

Reply:
Sure, I can change `sys.maxint` to `sys.float_info.max`; that said, I generally avoid assigning the maximum value of a type to a variable of that type (in any programming language) because if it is unintentionally increased, it will cause overflows (and becomes `inf` again in Python); e.g., think about `a`, `b` being max_float and we do `a+b`. It is not a huge concern here but are you okay with say max_float/2 (or even max_float/10), I think all of those are huge enough not to create any confusion that these are arbitrary "INF" values.

Reply:
Sure. I'm ok with max_float/10. Thanks!

Reply:
Done.

Reply:
1) That's correct, field name should change to IF. 
2) Hmm, not sure why overflow is a problem? If a+b makes an overflow (i.e., `inf`) in python, it is actually not bad as these numbers intended to be 'inf' in python?  It won't be a problem if the result is going to be exported back to BQ as the value converted again to max_float. What is the scenario you think overflow creates an issue?

Reply:
Nima, re. 2: As we discussed off-PR, there is not a very particular scenario, specially in this case, that I am concerned about overflows but a general habit/experience that setting values to the maximum of their type is not great because it increases the chance of downstream overflows/errors for reasons I mentioned (this is more of a concern when in some languages the overflow appears as a negative number).

In any case, if you have strong preference for using max_float directly, I can change the code that way, I don't think this is a big deal either way.